app/Http/Controllers/AuthenticateController.php

```diff
@@ -32,6 +32,17 @@ public function authenticate(Request $request){
                    ]);
   }  
 
+  //refresh route for the front end
+  public function user(){
+    $user = JWTAuth::parseToken()->toUser();
+    $token = JWTAuth::parseToken()->refresh(); 
+    $data = array();
+    $data["user"] = $user;
+    return response($data)->withHeaders([
+                      'Authorization'=> 'Bearer: '.$token
```

Comment:
Aight, all i'm saying is the spec for oauth states otherwise:

https://tools.ietf.org/html/rfc6749#section-4.4.3